index.html

```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<title>Page Title</title>
+	<link rel="stylesheet" type="text/css" media="screen" href="assets/style.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+		integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+</head>
+
+<body>
+	<header>
```

Comment:
broken div here,please when you give position that is leaving normal flow (absolute or fixed),than parent position can't be empty nor the size.

Reply:
Parent element of the div with the id ="header", now has a position, and a width and a height; We get now what you mean because our header tag was basically non-existent, and the div was overflowing his parent element.